crates/tiny/src/cmd.rs

```diff
@@ -313,16 +317,30 @@ fn join(args: CmdArgs) {
         }
     }
 
-    let words = args
-        .split_whitespace()
-        .map(|c| ChanName::new(c.to_owned()))
-        .collect::<Vec<_>>();
+    let chans = args
+        .split(',')
+        .map(str::trim)
+        .filter_map(|c| match Chan::from_str(c) {
+            Ok(c) => Some(c),
+            Err(err) => {
+                ui.add_client_err_msg(
+                    &format!("{}, Usage: {}", err, JOIN_CMD.usage),
+                    &MsgTarget::CurrentTab,
+                );
+                None
+            }
+        })
+        .collect::<Vec<Chan>>();
 
-    let chans: Vec<ChanName> = if words.is_empty() {
+    let chans = if chans.is_empty() {
         match ui.current_tab() {
             None => return,
+            // rejoin current tab channel
```

Comment:
"tab channel" sounds a bit strange to me. Maybe "rejoin current channel" or "rejoin current tab's channel"?